rmgpy/rmg/main.py

```diff
@@ -688,111 +687,113 @@ def execute(self, **kwargs):
                 
                 for index, reactionSystem in enumerate(self.reactionSystems):
                     
-                    reactionSystem.prunableSpecies = prunableSpecies
+                    reactionSystem.prunableSpecies = prunableSpecies   #these lines reset pruning for a new cycle
                     reactionSystem.prunableNetworks = prunableNetworks
+                    reactionSystem.reset_max_edge_species_rate_ratios() 
```

Comment:
Why is this needed in the new setup?

Reply:
This resets the maxEdgeSpeciesRateRatios array to zero, this isn't done in simulate anymore since we need to accumulate the values across all ranged reactor simulations.